api/hypercore/convert_to_rowstore.md

```diff
@@ -8,11 +8,15 @@ api:
   type: procedure
 ---
 
+import Deprecated2200 from "versionContent/_partials/_deprecated_2_20_0.mdx";
 import Since2180 from "versionContent/_partials/_since_2_18_0.mdx";
 import HypercoreManualWorkflow from "versionContent/_partials/_hypercore_manual_workflow.mdx";
 
 # convert_to_rowstore() <Tag type="community">Community</Tag>
 
+<Deprecated2200 /> TimescaleDB is optimized for fast updates on compressed data in the columnstore. To modify data in the 
```

Comment:
The API `convert_to_rowstore` is not deprecated. The approach to convert to the rowstore in order to do backfills is deprecated and users can just use standard SQL to do the job.

Reply:
This notice is best to be removed, it's misleading

Reply:
Done